### DISCUSSION TITLE
Outline for our first blogpost

### DISCUSSION THREAD

**Discussion by AakashKumarNain** [AakashKumarNain/diffusion_models#3] [Ideas]
We need to list down all the major things that we want to include in Part-1

**Comment by rish-16** [comment]
I feel it'd be a good move to have a primer on probability distributions (especially probability integrals, etc.) and statistics before we get into HOW we use them for diffusion models.

Also, we need a huge "glossary" of symbols and notations we'll be using for the article.

**Reply by AakashKumarNain** [reply]
Can we restrict the space to the terms that are used for diffusion models? We can always add other as we move on

**Reply by rish-16** [reply]
Definitely, Iet's focus on structure first then

**Reply by AakashKumarNain** [reply]
Here is a rough sketch:
1. Primer on `RV`, `Likelihood`, `Multi-variate Gaussian`, `Isotropic Gaussian`
2. Very short introduction to generative models along with the disadvantages
3. Reparameterization trick

Let me know what you think

**Comment by rish-16** [comment]
Sounds good, I can take care of random variables and more of the coding side of things. I'm not as familiar with the heavy probability math since I haven't studied that stuff in college yet 😅

**Reply by AakashKumarNain** [reply]
Haha no worries. I will take care of the rest of the terms. Let's aim to finish our first draft by next week

**Reply by rish-16** [reply]
Can I please have 2 weeks instead? Something important came up this week at work and I have to rush it 😅

**Reply by AakashKumarNain** [reply]
Sure. No rush at all
